Make <img> in 1-Index-html translatable

ignotus666 opened the pull request:
<!-- Thank you for working on Jamulus and opening a Pull Request! Please fill in the following to make the review process straightforward -->

**Short description of changes**
This adds an option to the po4a script to make the content of the `<img>` tag translatable - it was being skipped by po4a and made it impossible to show localised screenshots of the main mixer window on the homepage. It also makes it possible to easily add an arbitrary number of options in future if needed.

I made a small change to the EN version of 1-Index.md to trigger an update of the .po files. I think the change was necessary anyway as not everyone might have 5 users in their screenshot.

**Context: Fixes an issue? Related issues**
This problem was pointed out by @henkdegroot on Discord.

**Status of this Pull Request**
Tested and working

**What is missing until this pull request can be merged?**
Any Weblate PRs have to be merged first.


**Does this need translation?**

YES.


## Checklist
<!-- Please tick the check boxes when done by replacing the space by an x, e.g. [x]. -->
- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. <!-- GitHub doesn't run these checks for new contributors automatically. -->
- [x] I'm sure that this Pull Request goes to the correct branch

github-actions[bot] committed:
Merge release into next-release

github-actions[bot] committed:
Merge release into next-release

github-actions[bot] committed:
Merge release into next-release

github-actions[bot] committed:
Merge release into next-release

mcfnord committed:
Use newlines for code to help with copy-paste (#968)

* newlines help with copy-paste

* use tabs rather than 4 spaces

* three tics for copy button

* preferred markdown

github-actions[bot] committed:
AUTO: Updated .po files

ann0see committed:
Change jamulus-server to jamulus-headless (#969)

* Change jamulus-server to jamulus-headless

* Fix grammar

Co-authored-by: John Dempsey <1750243+mcfnord@users.noreply.github.com>

---------

Co-authored-by: John Dempsey <1750243+mcfnord@users.noreply.github.com>

github-actions[bot] committed:
AUTO: Updated .po files

ignotus666 committed:
Fix segmentation in 1-Index (#970)

* Update _config.yml

Make kramdown the markdown processor

* Update and rename 1-index.html to 1-index.md

* Update 1-index.md

* Update 1-de-index.html

* Rename 1-de-index.html to 1-de-index.md

* Update and rename 1-es-index.html to 1-es-index.md

* Update and rename 1-fr-index.html to 1-fr-index.md

* Update and rename 1-index.html to 1-index.md

* Update and rename 1-it-index.html to 1-it-index.md

* Update and rename 1-ko_KR-index.html to 1-ko_KR-index.md

* Update and rename 1-nb_NO-index.html to 1-nb_NO-index.md

* Update and rename 1-nl-index.html to 1-nl-index.md

* Update and rename 1-pt_BR-index.html to 1-pt_BR-index.md

* Update and rename 1-pt_PT-index.html to 1-pt_PT-index.md

* Update and rename 1-zh_CN-index.html to 1-zh_CN-index.md

* Fix segmentation in 1-Index

* Don't include processed .po files

* Correction

* Add comments

* Add comments

* Update _po4a-tools/po4a-update-templates.sh

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>

* Remove unneeded check

---------

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>

github-actions[bot] committed:
AUTO: Updated .po files

ignotus666 committed:
Revert "AUTO: Updated .po files"

This reverts commit a73218ec98da4229e23b724788ccaee682fad9b3.

ignotus666 committed:
Revert "Fix segmentation in 1-Index (#970)"

This reverts commit 256a67898cfdf9b5b5148f08edae645fbc17c455.

weblate committed:
Translations update from Hosted Weblate (#975)

* Update Spanish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/es/
Translation: Jamulus/Running-a-Server

* Update French translation using Weblate

Currently translated at 100.0% (165 of 165 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Olivier Humbert <trebmuh@tuxfamily.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/fr/
Translation: Jamulus/Directories
Translation: Jamulus/Running-a-Server

* Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/

---------

Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Co-authored-by: Olivier Humbert <trebmuh@tuxfamily.org>

ignotus666 committed:
Fix segmentation in 1-Index 2nd attempt (#976)

* Update _config.yml

Make kramdown the markdown processor

* Update and rename 1-index.html to 1-index.md

* Update 1-index.md

* Update 1-de-index.html

* Rename 1-de-index.html to 1-de-index.md

* Update and rename 1-es-index.html to 1-es-index.md

* Update and rename 1-fr-index.html to 1-fr-index.md

* Update and rename 1-index.html to 1-index.md

* Update and rename 1-it-index.html to 1-it-index.md

* Update and rename 1-ko_KR-index.html to 1-ko_KR-index.md

* Update and rename 1-nb_NO-index.html to 1-nb_NO-index.md

* Update and rename 1-nl-index.html to 1-nl-index.md

* Update and rename 1-pt_BR-index.html to 1-pt_BR-index.md

* Update and rename 1-pt_PT-index.html to 1-pt_PT-index.md

* Update and rename 1-zh_CN-index.html to 1-zh_CN-index.md

* Fix segmentation in 1-Index

* Don't include processed .po files

* Correction

* Add comments

* Add comments

* Update _po4a-tools/po4a-update-templates.sh

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>

* Remove unneeded check

* AUTO: Updated .po files

---------

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>
Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

github-actions[bot] committed:
AUTO: Updated .po files

github-actions[bot] committed:
Merge release into next-release

ann0see committed:
Add changes to iOS install page

weblate committed:
Translations update from Hosted Weblate (#978)

jujudusud committed:
Update Installation-for-Linux.md adding PipeWire usage. (#980)

github-actions[bot] committed:
AUTO: Updated .po files

ann0see committed:
Apply suggestion

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>

ignotus666 committed:
Update deprecated actions

ann0see committed:
Merge pull request #984 from ignotus666/dep-actions

Update deprecated actions

gilgongo committed:
Replace banner image

gilgongo committed:
Add figure style

ignotus666 committed:
Lang select (#11)

* Initial replacements testing OK so far

* Add language selection dropdown

* Don't store lang preference

* Align dropdown to the right. Remove unneeded code from fw.css

* Make dropdown scrollable

* Clean/move/improve css styling, add icon, lang sel. outline

* Remove unneeded css

* Show full lang names, automate adding langs

* AUTO: Add new language: (eu) #4

* Revert "AUTO: Add new language: (eu) #4"

This reverts commit 301c9486933621f8b5ae9300b4b7ba106b55662e.

* Correction

* AUTO: Add new language: (Basque) #5

* Update add-lang.yml

* AUTO: Add new language: (English (American)) #6

* Correction

* Revert language additions

* Correction

* Correction

* Update

* Update

* Add check

* Update error message

* Revert lang code hyphens

* Add changes by @gilgongo

* Combine with gilgongo's stuff

* Correct title check

* Correct pt-BR language codes

* Revert lang code changes

---------

Co-authored-by: Gilgongo <gilgongo@phreak.co.uk>
Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

ignotus666 committed:
Replace with simplified version

ignotus666 committed:
Change add-lang message, remove suglang reference

ignotus666 committed:
Improve accessibility

ignotus666 committed:
Match text colour

ignotus666 committed:
Make dropdown menu more compact

ignotus666 committed:
Make dropdown standard html

ignotus666 committed:
Replace with simplified version

Change add-lang message, remove suglang reference

Improve accessibility

Match text colour

Make dropdown menu more compact

Make dropdown standard html

ignotus666 committed:
Merge branch 'lang-select2' of https://github.com/ignotus666/jamuluswebsite into lang-select2

ignotus666 committed:
Add no-JS option

ignotus666 committed:
Correction to no-JS list

ignotus666 committed:
Make non-JS selection highlighted

ignotus666 committed:
Add language-container to fw.css

ignotus666 committed:
Correct translation readme

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/

ann0see committed:
Merge pull request #991 from weblate/weblate-jamulus-1-index

Translations update from Hosted Weblate

ann0see committed:
Rephrase sentence

Co-authored-by: Peter L Jones <pljones@users.noreply.github.com>

ignotus666 committed:
Fix canonical/alternate links

ignotus666 committed:
Fix canonical/alternate links

gilgongo committed:
Use class for figure

gilgongo committed:
Add class for home page img

gilgongo committed:
Better class name

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>

gilgongo committed:
Better class name

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>

softins committed:
Removed lrelease and .qm files check-in from release process.

These steps no longer needed as .qm files are now compiled on the fly.

ann0see committed:
Merge pull request #982 from jamulussoftware/update-ios-doc

Update iOS page with USBC and further information

ann0see committed:
Merge pull request #989 from jamulussoftware/homepage-img

Replace home page banner image

github-actions[bot] committed:
AUTO: Updated .po files

softins committed:
Update Release-Process.md

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>

softins committed:
Merge pull request #996 from softins/obsoleted-lrelease-and-qm

Removed lrelease and .qm files check-in from release process.

ignotus666 committed:
Make current language non-selectable in non-JS mode

ignotus666 committed:
Tidy up and fix non-selectable EN

ignotus666 committed:
Update _includes/headtags.html

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>

ann0see committed:
Merge pull request #994 from ignotus666/lang-select2

ann0see committed:
Merge branch 'next-release' into release-update

ann0see committed:
Merge pull request #997 from jamulussoftware/release-update

Update next-release with content of release

ignotus666 committed:
Change po folders to new lang codes

ann0see committed:
Merge pull request #998 from ignotus666/fix_lang_codes

Change po folders to new lang codes

dependabot[bot] committed:
Bump rexml from 3.2.8 to 3.3.2

Bumps [rexml](https://github.com/ruby/rexml) from 3.2.8 to 3.3.2.
- [Release notes](https://github.com/ruby/rexml/releases)
- [Changelog](https://github.com/ruby/rexml/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/rexml/compare/v3.2.8...v3.3.2)

---
updated-dependencies:
- dependency-name: rexml
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

ann0see committed:
Merge pull request #1000 from jamulussoftware/dependabot/bundler/rexml-3.3.2

Bump rexml from 3.2.8 to 3.3.2

ann0see committed:
Merge branch 'next-release' into release

ann0see committed:
Update dependencies

ann0see committed:
Merge pull request #1001 from jamulussoftware/release

Apply rexml upgrade again

ann0see committed:
Add connection setup and advanced setup screenshots

github-actions[bot] committed:
Merge release into next-release

ignotus666 committed:
Add connection setup and advanced setup screenshots in ES

ann0see committed:
Merge pull request #1004 from ann0see/screenshots/addScreenshotUpdates

ann0see committed:
Merge pull request #1006 from ignotus666/upd_scrnshts

pljones committed:
Mention connect via menu does not force disconnect

pljones committed:
Expand description of server list and say "directory"

pljones committed:
Add note about delete button

pljones committed:
Fix "My Profile" screenshot border

ann0see committed:
Merge pull request #1007 from pljones/issue-1002-delete-server-button-docs

github-actions[bot] committed:
AUTO: Updated .po files

ann0see committed:
Fix syntax error

ann0see committed:
Update dependencies

github-actions[bot] committed:
Merge release into next-release

github-actions[bot] committed:
Merge release into next-release

github-actions[bot] committed:
AUTO: Add new language: sv: Svenska #999

ignotus666 committed:
Revert "AUTO: Add new language: sv: Svenska #999"

This reverts commit 1eb8e58067e0a72ea959963445ec03f0a7266c3a.

github-actions[bot] committed:
AUTO: Add new language: sv-SE: Svenska #1022

henkdegroot committed:
update-nl-images-for-3-11

ann0see committed:
Merge pull request #1023 from henkdegroot/update-3-11-screenshots

ignotus666 committed:
Make <img> translatable

ignotus666 committed:
"Translatable" option not needed.

jujudusud commented:
I will be able to modify the french translation for "serveral" instead of "five" for the 3.11 release when I will add the screenshots.

pljones commented:
Do we need this for 3.11.0?

ignotus666 commented:
I suppose it's not essential, but merging it before the release won't do any harm either - it will make strings that aren't visible on the website anyway translatable. If people do manage to update the strings (it's just a couple, and only useful if they have localised screenshots for their language), then that's a bonus.

pljones commented:
It's more that I'm thinking we shouldn't bother adding jamulussoftware/jamuluswebsite changes to the Tracking board if they're not targeting a specific release, as that's what we're Tracking on the board.  So if this is just on going maintenance, that's fine.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Italian translation using Weblate

Currently translated at 100.0% (13 of 13 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/it/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Startpage

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Swedish translation using Weblate

Currently translated at 100.0% (16 of 16 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (27 of 27 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (31 of 31 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (56 of 56 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update Swedish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Co-authored-by: ignotus <ignotus666@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/client-troubleshooting/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/sv/
Translate-URL: https://hosted.weblate.org/projects/jamulus/unregistered-servers/sv/
Translation: Jamulus/Client-Troubleshooting
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Server-Bandwidth
Translation: Jamulus/Server-Troubleshooting
Translation: Jamulus/Software-Manual
Translation: Jamulus/Unregistered-Servers

weblate committed:
Update Spanish translation using Weblate

Currently translated at 100.0% (137 of 137 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ignotus <ignotus666@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/es/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Chinese (Simplified) translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/zh_Hans/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS

weblate committed:
Update Dutch translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (31 of 31 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (16 of 16 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (27 of 27 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Update Dutch translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Henk De Groot <weblate@hdegroot.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-ios/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-bandwidth/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/server-troubleshooting/nl/
Translate-URL: https://hosted.weblate.org/projects/jamulus/unregistered-servers/nl/
Translation: Jamulus/Directories
Translation: Jamulus/Installation-for-iOS
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Server-Bandwidth
Translation: Jamulus/Server-Troubleshooting
Translation: Jamulus/Unregistered-Servers

ann0see committed:
Merge pull request #1019 from weblate/weblate-jamulus-1-index

Translations update from Hosted Weblate

ann0see committed:
Replace bundle id

ann0see committed:
Merge pull request #1031 from jamulussoftware/macOS/changeBundleId

Change bundle ID to app.jamulussoftware.*

github-actions[bot] committed:
AUTO: Updated .po files

weblate committed:
Update German translation using Weblate

Currently translated at 100.0% (6 of 6 strings)

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-client-commands/de/
Translation: Jamulus/Include-Client-Commands

weblate committed:
Update Dutch translation using Weblate

Currently translated at 100.0% (49 of 49 strings)

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-linux/nl/
Translation: Jamulus/Installation-for-Linux

weblate committed:
Update Swedish translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/sv/
Translation: Jamulus/Running-a-Server

weblate committed:
Update Portuguese (Portugal) translation using Weblate

Currently translated at 100.0% (13 of 13 strings)

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/pt_PT/
Translation: Jamulus/Startpage

ann0see committed:
Merge pull request #1032 from weblate/weblate-jamulus-1-index

gilgongo committed:
Identify canonical and alternate URLs (#1026)

Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>

gilgongo committed:
Merge branch 'next-release' into img-trans

softins commented on wiki/en/misc/1-index.md:
```diff
@@ -29,7 +29,7 @@ Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone
     <figure class="mainbannerfig">
       <a href="wiki/Getting-Started">
-      <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected.">
+        <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected.">
```
This file needs rebasing due to the change in #1037. That should fix the conflict reported by GH below.

ignotus666 committed:
Resolve conflicts

ignotus666 committed:
Resolve conflict

ann0see commented:
@ignotus666 please open a new PR with your changes...

ignotus666 commented:
Yes, will do.
